### DISCUSSION TITLE
How do you insert Jinja code without {{% raw %}} escaping?

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#3233] [Q&A] [answered]
Hi @squidfunk,
here you have Jinja code

![image](https://user-images.githubusercontent.com/30607/142283874-d19818ca-775d-4dce-ab3e-8cc13c495d71.png)

that [you do not insert inside raw escaping](https://github.com/squidfunk/mkdocs-material/blob/master/docs/setup/setting-up-the-header.md?plain=1#L45-L50).

```
{% raw %}

{% extends "base.html" %}

{% block announce %}
  <!-- Add announcement here, including arbitrary HTML -->
{% endblock %}

{% endraw %}
```

How is it possible to have it in Material for MkDocs?

Thank you

**Comment by facelessuser** [comment, marked as answer]
MkDocs doesn't run Jinja2 on the Markdown content, so there is no need to raw escape. If you are using a Jinja2 Python Markdown extension, then maybe you would.

**Reply by aborruso** [reply]
> MkDocs doesn't run Jinja2 on the Markdown content, so there is no need to raw escape. If you are using a Jinja2 Python Markdown extension, then maybe you would.

thank you @facelessuser , this is my settings, I'm using these extensions: https://github.com/opendatasicilia/ods-mkdocs-material/blob/main/mkdocs.yml#L88-L122

Am I using Jinja2 extension?

**Reply by facelessuser** [reply]
No, but it looks like you are using a MkDocs plugin called `macros` which I believe uses Jinja2.
